api/client/proto/authservice.proto

```diff
@@ -269,16 +269,13 @@ message RequestStateSetter {
 // RequestID is the unique identifier of an access request.
 message RequestID { string ID = 1 [ (gogoproto.jsontag) = "id" ]; }
 
-// RotateResetPasswordTokenSecretsRequest is a request to rotate token secrets.
-message RotateResetPasswordTokenSecretsRequest {
-    string TokenID = 1 [ (gogoproto.jsontag) = "token" ];
-}
+// RotateUserTokenSecretsRequest is a request to rotate token secrets.
+message RotateUserTokenSecretsRequest { string TokenID = 1 [ (gogoproto.jsontag) = "token" ]; }
```

Comment:
Can you test backward compatibility here? I'm not sure if you can rename messages like this without breaking backward compatibility.

Reply:
@alex-kovoy @russjones I tested with an auth that included the new changes with an older proxy `v6.2.8` in cloud staging, and I tested each endpoints that were affected by the change and it worked as expected. 

The only endpoint I added was `RotateUserTokenSecrets`, and this will call the fallback `RotateResetPasswordTokenSecrets` if former returns not implemented.

The renaming of message name should not matter according to here: https://developers.google.com/protocol-buffers/docs/encoding#structure, I didn't change the field numbers/types and the json tags that would cause breakage.

Reply:
I'm a bit late to this party (saw the email updates), but Lisa is spot on: as long as we are consistent keeping field types, numbers and json tags, the renames are safe to do. JSON marshaling/unmarshaling is usually the weak link, specially for protos that we might persist.